app/services/nytimes_service.rb

```diff
@@ -10,6 +10,11 @@ def articles
     response_body = parse(connection.get)
     articles_data = response_body["results"]
     Article.create_articles(articles_data)
+    keyword_engine = KeywordEngine.new
+    Article.all.each do |article|
+      keywords = keyword_engine.generate_keywords(article)
+      article.update(keywords: keywords)
+    end
```

Comment:
I like that idea.  It seems like we're updating articles all over the place.  Since we're pulling the data from numerous sources, I wonder whether or not it's possible to update all of them at one time.